### ISSUE TITLE
All pages should have one `<h1>`

### ISSUE THREAD

**Issue by neel-a-mehta** [seattle-uat/civiform#1457]
For A11y and screen readers' sake, we should have exactly one `<h1>` tag on each page. The language selection page ("Please select your preferred language") for applicants is one example of a page that has no `<h1>` tag. 

Find any other pages in CiviForm that lack an `<h1>` tag, or have multiple, and fix them.

## Original tester's report

> ACTUAL RESULT: Missing header on CiviForm language page.
> EXPECTED RESULT: Every page should have a heading 1.

**Comment by neel-a-mehta** [comment]
Looks good. Nice check to throw errors if no `<h1>` is present.
